public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -955,10 +955,13 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
     /**
      *
      * @param panoramaId
+     * @param force: force to change pano, even if walking is disabled
      * @returns {setPano}
      */
-    self.setPano = function (panoramaId) {
-        if (!status.disableWalking) {
+    self.setPano = function (panoramaId, force) {
```

Comment:
It is Ok in Javascript to pass fewer arguments to a function. If you don't pass it, it will simply be `undefined`. This is why I first check in the setPano function whether `force` is undefined or not.